permissions/plugin-pipeline-cps-oras.yml

```diff
@@ -6,4 +6,6 @@ paths:
 developers:
 - "jonesbusy"
 issues:
-  - jira: 29725
```

Comment:
JIRA was mistakenly chosen on the RPU issue... Sorry

I will open a helpdesk ticket to remove the component

Reply:
deleted

Reply:
Thanks!